core/templates/dev/head/pages/learner_dashboard/learner_dashboard.html

```diff
@@ -469,6 +605,40 @@ <h1 class="stat-value"><[subscriptionsList.length]></h1>
       </button>
     </div>
   </script>
+
+  <script type="text/ng-template" id="modals/editorViewSuggestion">
```

Comment:
Is this the same as the suggestion modal in the editor? If not, why call it editorViewSuggestion (since this is a learner-facing dashboard)?